Package.swift

```diff
@@ -233,6 +242,16 @@ let package = Package(
     cxxLanguageStandard: .cxx14
 )
 
+if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
```

Comment:
not sure if this is gated by a tools version thats too old, but I think you should be able to use `Context.environment` instead of pulling in `Foundation`

Reply:
Didn't know that existed! I guess it's been there since Swift 5.6. But we should probably update this systematically across all apple/swiftlang repos, it appears in a _lot_ of places.